app/assets/javascripts/discourse/tests/integration/components/form-kit/form-test.gjs

```diff
@@ -137,6 +137,12 @@ module("Integration | Component | FormKit | Form", function (hooks) {
     await formApi.submit();
 
     assert.dom(".bar").hasText("2");
+
+    formApi.addError("bar", { title: "Bar", message: "error_foo" });
+    // assert on the next tick
+    setTimeout(() => {
```

Comment:
we shouldn't need this, but that's my fault, we can merge it like this, I will look later